styles.css

```diff
@@ -1 +1,780 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;600;700&family=Lato&family=Poppins&family=Roboto:wght@700&display=swap');
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  overflow-x: hidden;
+  font-family: 'Lato', sans-serif;
+  scroll-behavior: smooth;
+}
+
+main {
+  background-color: white;
+  background-image: url("./Images/background-opacity.png");
+  background-position: bottom;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+body {
+  background-color: rgba(211, 211, 211, 0.5);
+}
+
+.instructors-cards {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+.card2 {
+  gap: 5%;
+  width: 40vw;
+  margin: 2% 2% 2% 2%;
+  display: flex;
+  flex-direction: row;
+}
+
+.card2 img {
+  height: 25vh;
+  width: 35vw;
+  padding: 2% 2% 2% 2%;
+}
```

Comment:
As suggested by the previous reviewer, kindly make sure the speakers are displayed in two columns as provided in the design template.

Your design at 982px:
![image](https://user-images.githubusercontent.com/20554583/197352021-0ca8f673-4522-4479-815b-0937f37c51f6.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/197352130-50c975d1-e96c-476e-a6ea-3352a04c17d8.png)

Reply:
@emmiiorji  I don't know what's wrong on your end because I have tested it on my screen and on an online responsiveness website and it's working fine on both.
<img width="948" alt="Screenshot 2022-10-22 at 11 11 23 PM" src="https://user-images.githubusercontent.com/81438868/197356350-8847adb5-2bf5-45f7-84bc-40a892c5d9f1.png">
<img width="1278" alt="Screenshot 2022-10-22 at 11 14 27 PM" src="https://user-images.githubusercontent.com/81438868/197356424-da9d0e33-bbe6-4ec0-af9d-0348ce462f41.png">

You can cross-verify it by visiting this link.

https://responsivedesignchecker.com/checker.php?url=https%3A%2F%2Fwaseemqureshi2020.github.io%2FWD-Course.github.io%2F&width=920&height=768

Reply:
@emmiiorji  I don't know what's wrong on your end because I have tested it on my screen and on an online responsiveness website and it's working fine on both.
<img width="948" alt="Screenshot 2022-10-22 at 11 11 23 PM" src="https://user-images.githubusercontent.com/81438868/197356350-8847adb5-2bf5-45f7-84bc-40a892c5d9f1.png">
<img width="1278" alt="Screenshot 2022-10-22 at 11 14 27 PM" src="https://user-images.githubusercontent.com/81438868/197356424-da9d0e33-bbe6-4ec0-af9d-0348ce462f41.png">

You can cross-verify it by visiting this link.

https://responsivedesignchecker.com/checker.php?url=https%3A%2F%2Fwaseemqureshi2020.github.io%2FWD-Course.github.io%2F&width=920&height=768